# JCP300 - RTY
string firmware_version
string version_number
int32 last_time_run
int32 total_operation_time
string serial_number
string turbine_type